src/Agent/NewRelic/Profiler/Configuration/InstrumentationConfiguration.h

```diff
@@ -163,6 +163,7 @@ namespace NewRelic { namespace Profiler { namespace Configuration
             instrumentationPoint->ClassName = segments[1];
             instrumentationPoint->MethodName = segments[2];
             instrumentationPoint->Parameters = nullptr;
+            instrumentationPoint->TracerFactoryArgs = 0;
```

Comment:
Should we also set some default values for when this type is constructed?

Reply:
Yeah good idea